config.placeholder.yml

```diff
@@ -0,0 +1,11 @@
+default:
```

Comment:
The config package only accepts yaml:
https://cran.r-project.org/web/packages/config/vignettes/introduction.html

Alternatively, the Web API sample does not use the config library but direct variable assignments in a separate R script:
https://github.com/osisoft/sample-pi_web_api-common_actions-r/blob/main/test_config.placeholder.R

From what I can tell, the config package is what most people use; I agree that json would be better in general, but this felt to be the cleanest approach for now for R, so that's why I went with it.

Reply:
OK, that's fine I suppose. We can revisit it if/when we get around to trying to standardize on JSON. For now if yml makes the most sense for R it's probably what we should use.